tests/phpunit/tests/functions/isWpInstallationCompatible.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Tests the is_wp_installation_compatible() function.
+ *
+ * @group functions.php
+ * @covers ::is_wp_installation_compatible
+ */
+class Tests_Functions_IsWpInstallationCompatible extends WP_UnitTestCase {
+	/**
+	 * Tests is_wp_installation_compatible().
+	 *
+	 * @dataProvider data_is_wp_installation_compatible
+	 *
+	 * @param mixed $required The required WordPress installation, truthy for multisite and falsy for .
```

Comment:
Does anything missing for this? 

```suggestion
	 * @param mixed $required The required WordPress installation, true for multisite false otherwise.
```